slsim/Sources/SourceVariability/sinusoidal_variability.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+
+class SinusoidalVariability(object):
+    """This class manages sinusoidal variability of a source."""
+
+    def __init__(self, amp, freq):
+        """
+        :param amp: amplitude for a given source
+        :param freq: frequency of a given source
+        """
+        self.amp = amp
+        self.freq = freq
+
+    def magnitude(self, observation_time):
+        """Provides magnitude at a given.
```

Comment:
"at a given." given what?

Reply:
Thanks Nikki! It should be at a given observation time.